tests/serverpod_test_server/test/serialization_test.dart

```diff
@@ -17,6 +20,30 @@ void main() {
     expect(unpacked.aStringifiedEnum, isNull);
   });
 
+  test(
```

Comment:
Question: For my knowledge - What was the reason these tests where included here? Did you notice a discrepency with the serialization?

Reply:
I added these because I was investigating how to user `serverpod` types within models. But it seems it is not possible to use them on the client, only on the server as demonstrated in the tests here.

Meaning this feature of using `serverpod` types seem some what broken and I can't imagine that anyone is actually using it. The problem is that the generated types are not exported from the package.